src/App.js

```diff
@@ -1,14 +1,35 @@
 import { Routes, Route } from "react-router-dom";
+import { useId, useState } from "react";
 
 import Layout from "./components/Layout";
 import ErrorPage from "./pages/ErrorPage";
 import Home from "./pages/Home";
 import Login from "./pages/Login";
 import New from "./pages/New";
 import Customer from "./pages/Customer"
-import Order from "./pages/Order";
+import Order from "./pages/NewOrder";
```

Comment:
The file is called `NewOrder`, but the imported component is called `Order`. Consider naming your components the same way you named the component file in order to avoid confusing yourself and fellow developers.